blog/content/post/2021-10-10-optiplex-dasharo.md

```diff
@@ -0,0 +1,173 @@
+---
+title: 'Dasharo for Dell OptiPlex 7010 / 9010'
+abstract: 'Open source firmware may be hundred times better than the
+           proprietary one. On the example of Dell OptiPlex 7010 / 9010 we will
+           show you the advantages of Dasharo firmware on this machine.'
+cover: /covers/dasharo-sygnet.svg
+author: michal.zygowski
+layout: post
+published: true
+date: 2021-10-10
+archives: "2021"
+
+tags:
+  - firmware
+categories:
+  - Firmware
+
+---
+
+# Introduction
+
+Dell OptiPlex 7010 / 9010 is the majority of machines used in 3mdeb office by the
+engineers and developers. It is only natural for a company propagating open
+source firmware (OSFW) to utilize their own product in daily work. The process
+of porting coreboot on the machine took significant amount of free time (which
+you may read about in [other blog posts](https://blog.3mdeb.com/2020/2020-06-24-dell-optiplex-port/)
+by the way) but it was definitely worth it. It opened a path for developing a
+firmware that would squeeze out the full potential of the platform. You may be
+wondering what else a platform may do that OEM didn't implement. But I will
+tell you this, with open source firmware almost anything is possible and with
+[Dasharo firmware](https://dasharo.com/) we make it a reality.
+
+## ME neutralization
+
+Intel Management Engine is a very controversial component of modern Intel
+chipsets and SoCs due to its capabilities and features. One of the most fearful
+features is the remote management and connectivity. Intel ME is able to utilize
+network via dedicated network controller on vPro machines without any awareness
+of the operating system software. It is used for software like Intel Active
+Management Technology ([AMT](https://www.intel.co.uk/content/www/uk/en/architecture-and-technology/intel-active-management-technology.html))
+or [Computrace](https://i.dell.com/sites/content/business/solutions/brochures/en/Documents/absolute-overview.pdf).
```

Comment:
i don't know how, but I found references that Computrace is using Intel Anti-Theft Technology (AT-p) and  AT-p is a service runner by ME. This [whitepaper](https://media9.connectedsocialmedia.com/intel/06/4470/Intel_Anti_Theft_Technology_Computrace_WhitePaper.pdf) says:
```
Computrace,* a leading IT asset-management and security solution from Absolute® Software,  now takes advantage of Intel® Anti-Theft Technology – PC Protection (Intel® AT-p).
```
And here is a photo of an app that checks ME Management and Security Status showing AT-p as a ME service:
![scr01](https://user-images.githubusercontent.com/30656491/140996658-38157c4d-f7d0-4d1f-8d37-4950c1eae4bc.png)
For me this is enough to prove the relation

Reply:
@miczyg1 maybe providing link to that information, most probably using [wayback machine](https://web.archive.org/web/2020*/https://media9.connectedsocialmedia.com/intel/06/4470/Intel_Anti_Theft_Technology_Computrace_WhitePaper.pdf) in case of document disappearing in long run,  would be good addition to blog post.

Reply:
This page says it is discontinued for AT 1.x and 2.x, it doesn't say anything about newer versions thou @XenoKovah 

@pietrushnic again, that was not my point to elaborate on proofs here.

Reply:
Added the link to the picture and reference to the whitepaper.